ScreenshotTests/SnapshotHelper.swift

```diff
@@ -193,16 +193,16 @@ open class Snapshot: NSObject {
     }
 
     class func fixLandscapeOrientation(image: UIImage) -> UIImage {
-        #if os(watchOS)
-            return image
-        #else
+        if #available(iOS 10.0, *) {
```

Comment:
Since our minimum target version is v13.0, will this line logic branch ever be hit?